viadot/utils.py

```diff
@@ -83,29 +109,37 @@ def handle_api_response(
     params: Dict[str, Any] = None,
     headers: Dict[str, Any] = None,
     timeout: tuple = (3.05, 60 * 30),
+    method: Literal["GET", "POST", "DELETE"] = "GET",
+    data: str = None,
 ) -> requests.models.Response:
     """
     Handle an HTTP response by applying retries and handling some common response
     codes.
 
     Args:
-        url (str): the URL which trying to connect.
-        auth (tuple, optional): authorization information. Defaults to None.
-        params (Dict[str, Any], optional): the request parameters. Defaults to None.
-        headers: (Dict[str, Any], optional): the request headers. Defaults to None.
-        timeout (tuple, optional): the request times out. Defaults to (3.05, 60 * 30).
+        url (str): The URL which trying to connect.
+        auth (tuple, optional): Authorization information. Defaults to None.
+        params (Dict[str, Any], optional): The request parameters. Defaults to None.
+        headers: (Dict[str, Any], optional): The request headers. Defaults to None.
+        timeout (tuple, optional): The request times out. Defaults to (3.05, 60 * 30).
 
     Raises:
-        ReadTimeout: stop waiting for a response after `timeout` seconds.
+        ReadTimeout: Stop waiting for a response after `timeout` seconds.
```

Comment:
Timeout is a tuple, so this description is ambiguous. Timeout could be both 1 and 99999 seconds at the same time. But I think this should be ok if you fix `timeout` param docstring to explain which value is which.

Reply:
[📝 Updated docstrings in utils.py](https://github.com/dyvenia/viadot/pull/661/commits/ec9e1879f7155aa71233f9fad2aeba5560f290c0)